src/Skeleton/Main.java

```diff
@@ -220,18 +231,19 @@ static void readTeleportkapuk(Scanner reader) {
             String[] temp = line.split(" ");
             //0 aszteroida hibakezeles
             Aszteroida a1, a2;
+            Teleportkapu k1, k2;
             if (temp[1].equals("0")) {
-                a1 = null;
+                k1 = new Teleportkapu(Boolean.parseBoolean(temp[2]));
             } else {
                 a1 = (Aszteroida) NamesMap.get(temp[1]);
+                k1 = new Teleportkapu(a1, Boolean.parseBoolean(temp[2]));
```

Comment:
Itt már nem jó a parseboolean, ha jól emlékszem úgy döntöttünk marad a nemkergültmeg-megkergült

Reply:
Ezt gyűlésen nézzük meg. Nekem működött.

Reply:
Persze, ha csak null van, így is megy, lehet az volt mindig a tieidben